net/chrony/src/opnsense/mvc/app/controllers/OPNsense/Chrony/forms/general.xml

```diff
@@ -0,0 +1,40 @@
+<form>
+    <field>
+        <id>general.enabled</id>
+        <label>Enable</label>
+        <type>checkbox</type>
+        <help>Enable Chrony time daemon.</help>
+    </field>
+    <field>
+        <id>general.port</id>
+        <label>Listen Port</label>
+        <type>text</type>
+        <help>Set the port chrony listen to.</help>
+    </field>
+    <field>
+        <id>general.peer1</id>
+        <label>Peer 1</label>
+        <type>text</type>
+        <help>Set the primary NTP Peer.</help>
```

Comment:
since all instructions look the same, do you think a CSVListField would fit better to have as many servers as you like?

Reply:
HostnameField with asList, good idea :)